src/components/Task/Task.js

```diff
@@ -0,0 +1,13 @@
+import stylesTask from './Task.module.css'
+
+export const Task = ({text}) => {
+    return (
+        <div className={stylesTask.taskItem}>
+            <input type="checkbox"/>
+            <p>{text}</p>
+            <button>
+                <svg xmlns="http://www.w3.org/2000/svg" width="18" height="18"><path fill="#494C6B" fill-rule="evenodd" d="M16.97 0l.708.707L9.546 8.84l8.132 8.132-.707.707-8.132-8.132-8.132 8.132L0 16.97l8.132-8.132L0 .707.707 0 8.84 8.132 16.971 0z"/></svg>
```

Comment:
Since you've already got some experience with Material UI, would you wanna try replace this SVG with one of the icons from [MUI](https://mui.com/material-ui/icons/)?